.circleci/config.yml

```diff
@@ -387,8 +388,11 @@ jobs:
             environment:
               BUILD: cabal-v2
               HLINT_OPTIONS: lint
+              DISABLE_SDIST_BUILD: "yes"
               HLINT_TARGETS: core/src src test benchmark
             command: |
+              apt-get install -y ghc-8.10.2
+              apt-get install -y cabal-install-3.2
```

Comment:
getting below error without these:
Looking for binary [ghc] in PATH...
Not found.
Looking for binary [ghc] in [/opt]...
Not found.
Error: ghc  not found in PATH [/root/.local/bin:/root/.cabal/bin:/opt/ghc/bin:/opt/ghcjs/bin:/sbin:/usr/sbin:/bin:/usr/bin] or in /opt/ghc/*/bin

Need cabal-install because 'BUILD=cabal-v2'
Looking for binary [cabal] in PATH...
Not found.
Looking for binary [cabal] in [/opt]...
Not found.
Required command [cabal] not found in PATH [/root/.local/bin:/root/.cabal/bin:/opt/ghc/bin:/opt/ghcjs/bin:/sbin:/usr/sbin:/bin:/usr/bin].